.github/workflows/integration-tests-frontend.yml

```
@@ -0,0 +1,65 @@
+name: Integration Tests Frontend
+
+on:
+  workflow_call:
+    inputs:
+      target_branch:
+        required: true
+        type: string
+
+env:
+  TARGET_BRANCH: ${{ inputs.target_branch }}
+
+jobs:
+  integration_tests_frontend:
+    name: Cypress integration tests
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: "14"
+          cache: yarn
+          cache-dependency-path: frontend/yarn.lock
+
+      - name: Download go binary
+        uses: actions/download-artifact@v3
+        with:
+          name: kiali
+          path: ~/go/bin/
+
+      - name: Ensure kiali binary is executable
+        run: chmod +x ~/go/bin/kiali
+
+      - name: Download frontend build
+        uses: actions/download-artifact@v3
+        with:
+          name: build
+          path: frontend/build
+
+      - name: Setup kind
+        run: hack/setup-kind-in-ci.sh
+
+      - name: Setup Error Rates demo
+        run: hack/istio/install-error-rates-demo.sh -c "kubectl"
+
+      - name: Set kiali URL
+        run: |
+          KIALI_URL="http://$(kubectl get svc kiali -n istio-system -o=jsonpath='{.status.loadBalancer.ingress[0].ip}'):20001/kiali"
```

for similar reasons as I mention in my previous review:

```suggestion
          KIALI_URL="http://$(kubectl get svc -l app.kubernetes.io/name=kiali -n istio-system -o=jsonpath='{.status.loadBalancer.ingress[0].ip}'):20001/kiali"
```